notebooks/QuickStart.ipynb

```diff
@@ -60,49 +89,110 @@
    "id": "c4cfb3e2-6419-40b0-8452-563ee0b070cc",
    "metadata": {},
    "source": [
-    "## Create and activate server configuration\n",
+    "### d) Add server configuration to secrets\n",
+    "\n",
+    "The easiest way to configure one or more Colab notebooks to point to the same ApertureDB server is to put the JSON config into a Colab secret.  \n",
+    "This secret will persist over kernel restarts and can be used by all of your notebooks, \n",
+    "but it will not be exposed if you share your notebook with anyone.\n",
+    "\n",
+    "![Screenshot of setting a Colab Secret](images/colab-secret.png)\n",
```

Comment:
I feel adding these steps to all the notebooks will make them too big. This should be added to setup notebooks colab section. https://docs.aperturedata.dev/Setup/client/notebooks .

Reply:
am not putting them in all - putting a reference to this notebook though. Quick start - self contained - get people going. others - we assume they are set up already.

Reply:
If I put this information in [/Setup/client/notebooks](https://docs.aperturedata.dev/Setup/client/notebooks) , should it augment or replace the instructions already there?

Reply:
Those should be replaced.